tests/consensus_tests/auth_tests/test_jwt_access.py

```
@@ -769,7 +769,7 @@ def get_auth_grpc_clients() -> GrpcClients:
 
 
 def check_access(
-    action_name: str, rest_request=None, grpc_request=None, path_params={}, rest_req_kwargs={}
+        action_name: str, rest_request=None, grpc_request=None, path_params={}, rest_req_kwargs={}
```

Ruff (0.14.1)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)